pkg/internal/apis/config/types.go

```diff
@@ -130,3 +126,110 @@ type PatchJSON6902 struct {
 	// Patch should contain the contents of the json patch as a string
 	Patch string
 }
+
+// Mount specifies a host volume to mount into a container.
```

Comment:
these are copied from pkg/container/cri/types.go, but the struct tags have been removed (for the internal API version)